edge_computing/day_2_core_cnf_clusters/telco-day-2-welcome.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="telco-day-2-welcome"]
+= Day 2 operations for telco core CNF clusters
+include::_attributes/common-attributes.adoc[]
+:context: telco-day-2-welcome
+
+toc::[]
+
+What follows is an overview of Day 2 operations you can use to manage telco core CNF clusters.
```

Comment:
Maybe a little more direct/less passive phrasing here?
```suggestion
You can use the following Day 2 operations to manage telco core CNF clusters.
```

Reply:
Thanks for the review @jeana-redhat 
Good suggestion